library/src/webapp/editor/ckextraplugins/common-wl/js/get-user-data.js

```diff
@@ -32,12 +15,12 @@ var getAvailableSites = function() {
 
       for (i in data) {
         var site = data[i];
-        sites.push([site.title, '/group/' + site.entityId + '/']);
+        if (site.id==currentSiteId){
+            sites.push([site.title, groupPath + site.entityId + forwardSlash]);
```

Comment:
This will only work where the user is a member of the current site. If they have access to a site through one of the large groups there will be no site that matches.